extensions/amp-story/1.0/amp-story-interactive-poll.css

```diff
@@ -0,0 +1,123 @@
+  box-shadow: var(--i-amphtml-interactive-chip-shadow-inset), inset 0px 0px 0px 1px var(--i-amphtml-interactive-theme-border) !important;
+  -webkit-box-shadow: var(--i-amphtml-interactive-chip-shadow-inset), inset 0px 0px 0px 1px var(--i-amphtml-interactive-theme-border) !important;
+  -moz-box-shadow: var(--i-amphtml-interactive-chip-shadow-inset), inset 0px 0px 0px 1px var(--i-amphtml-interactive-theme-border) !important;
+  animation: flashBGColor forwards var(--i-amphtml-interactive-animation-time) var(--i-amphtml-interactive-ease-out-curve) !important;
+}
+
+.i-amphtml-story-interactive-option::after {
+  content: '' !important;
+  background-color: var(--i-amphtml-interactive-theme-shading) !important;
+  position: absolute !important;
+  height: 100% !important;
+  width: 100% !important;
+  left: 0 !important;
+  top: 0 !important;
+  transform: translateX(-100%) !important;
+  transition: transform var(--i-amphtml-interactive-animation-time) var(--i-amphtml-interactive-ease-out-curve) !important;
+}
+
+.i-amphtml-story-interactive-poll-container[dir="rtl"] .i-amphtml-story-interactive-option::after {
+  transform: translateX(100%)!important;
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-option::after {
+  transform: translateX(calc(var(--option-percentage) - 100%)) !important;
+}
+
+.i-amphtml-story-interactive-poll-container.i-amphtml-story-interactive-post-selection[dir="rtl"] .i-amphtml-story-interactive-option::after {
+  transform: translateX(calc(100% - var(--option-percentage)))!important;
+}
+
+.i-amphtml-story-interactive-option-text {
+  font-size: 22px !important;
+  margin: 0 !important;
+  z-index: 1 !important;
+  transition: transform var(--i-amphtml-interactive-animation-time) var(--i-amphtml-interactive-ease-out-curve) !important;
+  transform-origin: left !important;
+}
+
+.i-amphtml-story-interactive-poll-container[dir="rtl"] .i-amphtml-story-interactive-option-text{
+  transform-origin: right !important;
+}
+
+.i-amphtml-story-interactive-post-selection .i-amphtml-story-interactive-option-text {
+  transform: scale(0.72) !important;
+}
+
+.i-amphtml-story-interactive-option-percentage {
+  font-weight: 700 !important;
+  font-size: 28px !important;
+  margin: 0 !important;
+  opacity: 0 !important;
+  transition: transform var(--i-amphtml-interactive-animation-time) var(--i-amphtml-interactive-ease-out-curve) !important;
+  transform: translateX(60px) !important;
+  direction: ltr;
+}
+
+.i-amphtml-story-interactive-poll-container[dir="rtl"] .i-amphtml-story-interactive-option-percentage {
+  transform: translateX(-60px) !important;
+}
+
+.i-amphtml-story-interactive-poll-container.i-amphtml-story-interactive-post-selection[dir="rtl"] .i-amphtml-story-interactive-option-percentage,
+.i-amphtml-story-interactive-poll-container.i-amphtml-story-interactive-post-selection:not([dir="rtl"]) .i-amphtml-story-interactive-option-percentage {
```

Comment:
Need to have stronger preference over the rules above, so they had to be defined very specifically